daemon/delete.go

```diff
@@ -141,9 +142,11 @@ func (daemon *Daemon) commonRm(container *Container, forceRemove bool) (err erro
 		return fmt.Errorf("Driver %s failed to remove root filesystem %s: %s", daemon.driver, container.ID, err)
 	}
 
-	initID := fmt.Sprintf("%s-init", container.ID)
-	if err := daemon.driver.Remove(initID); err != nil {
-		return fmt.Errorf("Driver %s failed to remove init filesystem %s: %s", daemon.driver, initID, err)
+	if runtime.GOOS != "windows" {
+		initID := fmt.Sprintf("%s-init", container.ID)
+		if err := daemon.driver.Remove(initID); err != nil {
```

Comment:
I see the issue here.  The Windows graph driver tries to remove some additional files, but returns error if those files do not exist.  This breaks the previous contract of the graph driver treating remove of a non-existent ID as a no-op.  The right thing to do here is fix the graph driver to correctly return no error, and leave this code as is.  John, you can feel free to delete this PR, I don't think this change will be needed long term.